src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -22,8 +22,8 @@ private void TestWindowApisUnix()
             // This tests the hardcoded implementations of these APIs on Unix.
             using (Process p = Process.GetCurrentProcess())
             {
-                Assert.True(p.Responding);
-                Assert.Equal(string.Empty, p.MainWindowTitle);
+                Assert.Throws<PlatformNotSupportedException>(() => p.Responding);
```

Comment:
Do we want these to change? Previously we were returning true and empty string, respectively. I would tend to think those are better than throwing -- it's resonable to assume that whatever library is calling these would happily continue with those return values.